src/Okta.Sdk/Api/RoleApi.cs

```diff
@@ -62,7 +62,7 @@ public interface IRoleApiAsync : IApiAccessor
         /// <param name="instance"></param>
         /// <param name="cancellationToken">Cancellation Token to cancel the request.</param>
         /// <returns>Task of IamRole</returns>
-        System.Threading.Tasks.Task<IamRole> CreateRoleAsync(  IamRole instance , System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken));
+        System.Threading.Tasks.Task<IamRole> CreateRoleAsync(  CreateIamRoleRequest instance , System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken));
```

Comment:
nitpick: I feel like it would be preferred for parameters that are some type of request object be named `request`.

Reply:
I followed the same convention as existing APIs for consistency.